index.html

```diff
@@ -2458,14 +2458,24 @@ <h3>Data Schemas</h3>
 
         <dl>
           <dt><var>credentialSchema</var></dt>
-          <dd>
-The value of the <code>credentialSchema</code> <a>property</a> MUST be one or
-more data schemas that provide <a>verifiers</a> with enough information to
-determine if the provided data conforms to the provided schema. Each
-<code>credentialSchema</code> MUST specify its <code>type</code> (for example,
-<code>JsonSchemaValidator2018</code>), and an <code>id</code> <a>property</a>
-that MUST be a <a>URL</a> identifying the schema file. The precise contents of
-each data schema is determined by the specific type definition.
+          <dd><p>
+The value of the <code>credentialSchema</code> <a>property</a> MUST be a single
+data schema object which provides <a>verifiers</a> with enough information 
+to determine if the provided data conforms to the provided schema object. Each
+<code>credentialSchema</code> MUST specify its <code>type</code> (e.g.,
+<code>JsonSchemaValidator2018</code>, 
+<code><a href="https://w3c-ccg.github.io/vc-json-schemas/">CredentialSchema2022</a></code>,
+or <code><a href="https://transmute-industries.github.io/vc-credential-schema-open-api-specification/">
+OpenApiSpecificationValidator2022</a></code>), and an <code>id</code> <a>property</a>
+which MUST be a <a>URL</a> identifying the schema file. </p>
+
+<p>The precise contents of each data schema object is determined by its 
+<code>type</code> definition. Different <code>type</code> implementations
+of data schema objects may choose to support different processing mechanisms 
```

Comment:
```suggestion
of data schema objects choose to support different processing mechanisms 
```

Reply:
Avoid using normative language such as "MUST/SHOULD/MAY" as lowercase text. Inevitably reviewers get confused and ask the Editors: "Do you mean 'MAY' here?" or worse "You're not testing this normative MUST requirement, it's lowercase, you need to make it uppercase." We've discovered that over time, that eats into Editor's time maintaining the spec, so we try to avoid lowercase normative language if at all possible by using "expected to/might/can" and other language that can't be confused with a normative statement.